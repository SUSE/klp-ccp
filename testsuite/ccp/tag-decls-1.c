struct A;

void pu_f(struct A*)
{}
