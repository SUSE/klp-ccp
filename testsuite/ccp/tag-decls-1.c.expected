/* klp-ccp: from testsuite/ccp/tag-decls-1.c */
struct A;

void klpp_pu_f(struct A*)
{}
